app/views/LargeUnsealedGoodsRecordDiscrepanciesYesNoView.scala.html

```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.MovementReferenceNumber
+@import viewModels.components.InputYesNoViewModel._
+@import viewModels.components.InputRadioViewModel._
+@import views.html.components.{Button, ErrorSummary, InputRadio}
+@import views.utils.ViewUtils.StringImplicits
+
+@this(
+mainTemplate: MainTemplate,
+formHelper: FormWithCSRF,
+inputRadio: InputRadio,
+errorSummary: ErrorSummary,
+button: Button
+)
+
+@(form: Form[Boolean],mrn: MovementReferenceNumber, arrivalId: ArrivalId, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("largeUnsealedGoodsRecordDiscrepanciesYesNo.title"),
+    showBackLink = true
+) {
+
+        @errorSummary(form.errors)
+
+        @formHelper(action = controllers.routes.LargeUnsealedGoodsRecordDiscrepanciesYesNoController.onSubmit(arrivalId, mode), Symbol("autoComplete") -> "off") {
+
+
+        @inputRadio(
+            field = form("value"),
+            radioType = RadioWithAdditionalHtml(
+                heading = messages("largeUnsealedGoodsRecordDiscrepanciesYesNo.heading"),
+                caption = Some(messages("site.caption", mrn.toString)),
+                additionalHtml = messages("largeUnsealedGoodsRecordDiscrepanciesYesNo.paragraph").toParagraph
+            ),
+            radioItems = fieldId => {
+            Seq(
+                RadioItem(
+                id = Some(fieldId),
+                value = Some("true"),
+                content = Text(messages("largeUnsealedGoodsRecordDiscrepanciesYesNo.yes")),
+                checked = form("value").value.contains("true")
+                ),
+            RadioItem(
+                id = Some(s"${fieldId}-no"),
+                value = Some("false"),
+                content = Text(messages("site.no")),
+                checked = form("value").value.contains("false")
+                )
+            )
+            }
```

Comment:
What is the reasoning behind not using the `InputYesNo` component?

Reply:
Ah I see, the yes message is different.

Why not just update the `InputYesNo` params to include `yesMessage: String = "site.yes"`

Then you can override it in your `LargeUnsealedGoodsRecordDiscrepanciesYesNoView` to use `largeUnsealedGoodsRecordDiscrepanciesYesNo.yes`